src/main/docker/cypress/Cypress-Dockerfile

```diff
@@ -1,5 +1,4 @@
 ARG CYPRESS_BROWSER=node18.6.0-chrome105-ff104
 FROM cypress/browsers:${CYPRESS_BROWSER}
-RUN export DEBIAN_FRONTEND=noninteractive && \
-    apt update && \
-        apt -y install openjdk-17-jdk
+RUN apt-get update
```

Comment:
Why not run _apt-get update, install, clean and removing apt_ install files in one step like in the `Coverage-Dockerfile`?

Reply:
Will do